show/main.py

```diff
@@ -1384,7 +1384,12 @@ def runningconfiguration():
 def all(verbose):
     """Show full running configuration"""
     cmd = "sonic-cfggen -d --print-data"
-    run_command(cmd, display_cmd=verbose)
+    output = json.loads(run_command(cmd, display_cmd=verbose, return_cmd=True))
+    if not multi_asic.is_multi_asic():
+        bgpraw_cmd = "sudo {} -c 'show running-config'".format(constants.RVTYSH_COMMAND)
```

Comment:
>sudo [](http://example.com/codeflow?start=22&length=4)

Could you check if really need `sudo`? #Closed

Reply:
I think if we use `rvtysh` we dont need need `sudo`. I think if we use sudo then the user has have RW privilages